Tests/WitsmlExplorer.Api.Tests/Extensions/HttpRequestExtensionsTests.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Text;
+
+using Microsoft.AspNetCore.Http;
+
+using Moq;
+
+using WitsmlExplorer.Api.Configuration;
+using WitsmlExplorer.Api.Extensions;
+
+using Xunit;
+
+namespace WitsmlExplorer.Api.Tests.Extensions
+{
+    public class HttpRequestExtensionsTests
+    {
+        private readonly ServerCredentials _scInput;
+        public HttpRequestExtensionsTests()
+        {
+            _scInput = new()
+            {
+                Host = new Uri("http://validstring.com"),
+                Creds = new BasicCredentials()
+                {
+                    UserId = "username",
+                    Password = "password"
+                }
+            };
+        }
+
+        [Fact]
+        public void GetWitsmlServerHttpHeader_ValidInputWCreds_ServerCredentialsCorrectlyParsed()
```

Comment:
ValidInputWCreds - does the W stand for With?

Reply:
Yes. The method signature was getting wide. But I guess that is a common trait of test methods. 
I have no big preferences, but expanding the `W` to `With` does not make a big impact on length so I can change this